### PR TITLE
Solve Leetcode problems

### PR DESCRIPTION
**Pull request by maheshmnj**

### CODE REVIEW

**Comment by maheshmnj** [comment]
@saurabh47 Can we merge these before this PR gets long? or Should I continue? I intend to submit more solutions though.
All the solutions are passing the test cases on leetcode.

File: leetcode/problem_242.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            else:
+                freq_s[letter_i] = 1
+            if(letter_k in freq_k):
+                freq_k[letter_k] +=1
+            else:
+                freq_k[letter_k] = 1
+        result = True
+        for key,value in freq_s.items():
+            if(key not in freq_k or freq_s[key]!=freq_k[key]):
+                result = False
```
Comment (saurabh47) [inline leetcode/problem_242.py:23]: directly you can return False

File: leetcode/problem_242.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                break;
+        return result
```
Comment (saurabh47) [inline leetcode/problem_242.py:25]: directly you can return True

No need of extra variable

**Review by saurabh47 — APPROVED** [review]

File: leetcode/problem_242.py (hunk 1 above)
Comment (maheshmnj) [reply leetcode/problem_242.py]: resolved

File: leetcode/problem_242.py (hunk 2 above)
Comment (maheshmnj) [reply leetcode/problem_242.py]: resolved.

**Comment by maheshmnj** [comment]
This is ready for review @saurabh47

**Pull request merged**